<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowClass</class>
 <widget class="QMainWindow" name="mainwindowClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>IMDb Comparator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/icons/32x32/Resources/icon/32x32/IMDb-Comparator.ico</normaloff>:/icons/32x32/Resources/icon/32x32/IMDb-Comparator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#282828;
}

QMenuBar{
	background-color:#2d2d2d;
}

QMenuBar::item{
	background-color:#2d2d2d;
	color:#fff;
}

QMenuBar::item:selected{
	background: #404040;
}

QMenu{
	background-color:#1a1a1a;
	color:#fff;
	padding:0;
}

QMenu::item:selected{
	background-color:#343434;
	color:#fff;
}

QSplitter::handle{
    
}

QStatusBar{
	background-color:#2d2d2d;
	color:#fff;
}

</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Film Tables</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="100,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_4">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="opaqueResize">
               <bool>true</bool>
              </property>
              <property name="handleWidth">
               <number>4</number>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QGroupBox" name="groupBox_1">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>0</height>
                </size>
               </property>
               <property name="acceptDrops">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="title">
                <string>#1 .csv file</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QTableView" name="tableView_1">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="sizeAdjustPolicy">
                   <enum>QAbstractScrollArea::AdjustIgnored</enum>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>0</height>
                </size>
               </property>
               <property name="acceptDrops">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>#2 .csv file</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QTableView" name="tableView_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <item>
               <property name="text">
                <string>Show All</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Remove Duplicates</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Only Show Duplicates</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_featureFilms">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Show Only Feature Films</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Information</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QGroupBox" name="groupBox_infoPage_1">
              <property name="title">
               <string>#1 .csv file</string>
              </property>
              <widget class="QLabel" name="label_numHrs_1">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>20</y>
                 <width>200</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>28</pointsize>
                </font>
               </property>
               <property name="text">
                <string>NUM_HRS</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_1b">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>50</y>
                 <width>150</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <weight>50</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Film hours on record</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_filter_1">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>70</y>
                 <width>250</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>28</pointsize>
                </font>
               </property>
               <property name="text">
                <string>FILTER_NUM_HRS</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_filter_1b">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>100</y>
                 <width>200</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <weight>50</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Film (Filter) hours on record</string>
               </property>
              </widget>
              <zorder>label_numHrs_1b</zorder>
              <zorder>label_numHrs_1</zorder>
              <zorder>label_numHrs_filter_1</zorder>
              <zorder>label_numHrs_filter_1b</zorder>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_infoPage_2">
              <property name="title">
               <string>#2 .csv file</string>
              </property>
              <widget class="QLabel" name="label_numHrs_2">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>20</y>
                 <width>200</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>28</pointsize>
                </font>
               </property>
               <property name="text">
                <string>NUM_HRS_2</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_2b">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>50</y>
                 <width>150</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <weight>50</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Film hours on record</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_filter_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>70</y>
                 <width>250</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>28</pointsize>
                </font>
               </property>
               <property name="text">
                <string>FILTER_NUM_HRS_2</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_numHrs_filter_2b">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>100</y>
                 <width>200</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <weight>50</weight>
                 <italic>true</italic>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Film (Filter) hours on record</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuLoad">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionLoad_File_1"/>
    <addaction name="actionLoad_File_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLoad"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionLoad_File_1">
   <property name="text">
    <string>Load File #1</string>
   </property>
  </action>
  <action name="actionLoad_File_2">
   <property name="text">
    <string>Load File #2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
